app/javascript/lib/search-metrics.js

```diff
@@ -0,0 +1,275 @@
+import camelcaseKeys from 'camelcase-keys'
+
+import { getNumFacetsAndFilters } from '~/providers/StudySearchProvider'
+import { log } from '~/lib/metrics-api'
+
+let prevNumSearches = -1
```

Comment:
```suggestion
```
I think you can get rid of `prevNumSearches` if instead you reset `numSearchSelections` inside `logSearch`

Reply:
Elegant, confirmed locally (with a minor adjustment, ad0ad70).